docs/dictionary/property/currentCard.lcdoc

```diff
@@ -0,0 +1,36 @@
+Name: currentCard
+
+Type: property
+
+Syntax: the currentCard
+
+Summary:
+A stack property that is set to the short name of the card in a given stack that is selected for display.
+
+OS: mac, windows, linux, ios, android
+
+Platforms: desktop, server, mobile
+
+Example:
+ -- the following results in the same value as put the short name of the current card of this stack
+ 
+put the currentCard of this stack
+
+ -- even if the stack is invisible, in the background, or offscreen
+
+set the visible of this stack to false
+
+put currentCard of this stack
+
+-- can be retrieved or set for a stack file that is not currently opened (provided the stack file is writeable)
+
+set currentCard of stack "/Users/admin/Desktop/example.livecode" to "ExampleOfCardNameThatsNotTheFirstCard"
+
+Description:
+<currentCard> <property> is a stack property that is set to the short name of the card that is currently selected
+
+for display in a given stack, not neccessarly the front most stack nor necessarily a visible stack or even open stack.
```

Comment:
- Lines here shouldn't be double-spaced unless they are intended to be separate paragraphs.
- A standard form for Description elements is something like
```
Description:
Use the <currentCard> property to change the <current card> of a stack, or find out which card is the <current card> of a stack.
```
Then maybe you could add something about how the target stack can be not necessarily the frontmost, in contrast to <this> card, perhaps?